src/qibo/models/error_mitigation.py

```diff
@@ -101,10 +105,11 @@ def ZNE(
     observable,
     noise_levels,
     noise_model=None,
-    nshots=int(1e4),
+    nshots=10000,
     solve_for_gammas=False,
     insertion_gate="CNOT",
     readout: dict = {},
```

Comment:
On top of the review I just received from @andrea-pasquale, it would be better to set `None` as default value of a dictionary instead of a mutable variable. Then you can do:

```python
if readout is None:
    readout = {}
```

Take this comment valid for all the readout arguments that follow.